jpa/README.md

```diff
@@ -5,9 +5,9 @@ Java Persistence API (JPA) is a Java application programming interface specifica
 ## All Niche Things about using JPA
 
 | Title                                              | Description                                                                                                                                                                                                                                                                                                                                               |
-|----------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| [Custom Sequences](./boot-data-customsequence)     | <ol> Implements CustomSequence Generation like `ORD000001` </ol> <ol> Uses [Custom Repository](https://vladmihalcea.com/basejparepository-hypersistence-utils/) instead of OOTB spring data repository  </ol><ol>Logging SQL Statements using `DataSourceProxy` </ol> <ol> Connections Acquiring Optimization using `LazyConnectionDataSourceProxy` </ol> |
-| [Entity Auditing using envers](./boot-data-envers) |                                                                                                                                                                                                                                                                                                                                                           |
+|----------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Custom Sequences](./boot-data-customsequence)     | <ul> Implements CustomSequence Generation like `ORD000001` </ul> <ul> Uses [Custom Repository](https://vladmihalcea.com/basejparepository-hypersistence-utils/) instead of OOTB spring data repository  </ul><ul>Logging SQL Statements using `DataSourceProxy` </ul> <ul> Connections Acquiring Optimization using `LazyConnectionDataSourceProxy` </ul> |
+| [Entity Auditing using envers](./boot-data-envers) | Version History and Auditing of db using spring data envers based on Hibernate Envers                                                                                                                                                                                                                                                                     |
```

Comment:
@coderabbitai, can you please help me with updating this READMe.md file with information of all modules and submodules present in this folder.in the existing tabular format

Reply:
@coderabbitai, Thank you for helping me